### PR TITLE
fix: Pass Properties to subsequent stubs to honour security settings.

### PR DESCRIPTION
**Pull request by pgharron**
- Honour System Properties.

### CODE REVIEW

**Comment by pgharron** [comment]
Hi @skyghis , me again!  I've had to make another minor change as I realised server pings do not have the higher level of integrity required and this leads to lots of Windows system error events.
Thank you :+1: 
Phil

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
```
Comment (breunouddanedeu) [inline CHANGELOG.md:7]: Should add `### Fixes`.
See https://keepachangelog.com/en/1.1.0/

Comment (pgharron) [reply CHANGELOG.md:7]: Now updated.

**Review by skyghis — APPROVED** [review]

**Comment by skyghis** [comment]
@pgharron I created a minor release and maven deployment is in progress.

You should probably fork this repo and deploy your own version on maven central.

**Comment by pgharron** [comment]
> @pgharron I created a minor release and maven deployment is in progress.
> 
> You should probably fork this repo and deploy your own version on maven central.

@skyghis  Many thanks!

**Comment by pgharron** [comment]
> You should probably fork this repo and deploy your own version on maven central.

@skyghis Is this because this repo is no longer being actively maintained?

**Comment by skyghis** [comment]
> @skyghis Is this because this repo is no longer being actively maintained?

Yes

**Pull request merged**